### PR TITLE
Fix external sources

### PR DESCRIPTION
**Pull request by neonbunny**
This fixes both the 404'ing Azure source and the partial stackpath source

### CODE REVIEW

File: cmd/generate-index/provider.yaml [hunk 1]
```diff
@@ -18,6 +16,8 @@ cdn:
       - https://api.fastly.com/public-ip-list
     google:
       - https://www.gstatic.com/ipranges/goog.json
+    stackpath:
+      - https://k3t9x2h3.map2.ssl.hwcdn.net/ipblocks.txt
```
Comment (ehsandeep) [inline cmd/generate-index/provider.yaml:20]: @neonbunny do we know the source of this url / hosted by?

Comment (neonbunny) [reply cmd/generate-index/provider.yaml:20]: It's from the stackpath support pages - see comment on commit 4783aa5e71636c84b780c159ee206d0625f52db4

**Review by ehsandeep — APPROVED** [review]

File: cmd/generate-index/provider.yaml [hunk 2]
```diff
@@ -44,7 +44,7 @@ cloud:
     oracle:
       - https://docs.oracle.com/en-us/iaas/tools/public_ip_ranges.json
     azure:
-      - https://download.microsoft.com/download/7/1/D/71D86715-5596-4529-9B13-DA13A5DE5B63/ServiceTags_Public_20221128.json
+      - https://www.microsoft.com/en-us/download/confirmation.aspx?id=56519
```
Comment (tarunKoyalwar) [inline cmd/generate-index/provider.yaml:47]: @neonbunny , above link seems to be broken
```console
$  curl -I "https://www.microsoft.com/en-us/download/confirmation.aspx?id=56519" 
HTTP/2 302 
content-length: 0
location: https://www.microsoft.com/en-us/download/404Error.aspx
```

**Review by tarunKoyalwar — CHANGES_REQUESTED** [review]
```console
$  ./generate-index   
2023/05/09 02:47:14 [err] could not fetch cloud item: could not get url https://www.microsoft.com/en-us/download/confirmation.aspx?id=56519: Get "": unsupported protocol scheme ""
[common/fqdn] Defined 1 items for cloudflare
[common/fqdn] Defined 1 items for fastly
[common/fqdn] Defined 1 items for edgecast
[common/fqdn] Defined 1 items for incapsula
[common/fqdn] Defined 2 items for amazon
[common/fqdn] Defined 4 items for akamai
[cdn] Got 142 items for cloudfront
[cdn] Got 19 items for fastly
[cdn] Got 76 items for google
[cdn] Got 135 items for stackpath
[waf] Got 15 items for cloudflare
[waf] Got 11 items for incapsula
[cloud] Got 7324 items for aws
[cloud] Got 563 items for google
[cloud] Got 578 items for oracle
```

File: cmd/generate-index/provider.yaml (hunk 2 above)
Comment (ehsandeep) [reply cmd/generate-index/provider.yaml:47]:
```console
curl -I "https://www.microsoft.com/en-us/download/confirmation.aspx?id=56519" 
HTTP/2 200 
cache-control: no-cache, no-store
pragma: no-cache
content-type: text/html
expires: -1
p3p: CP="ALL IND DSP COR ADM CONo CUR CUSo IVAo IVDo PSA PSD TAI TELo OUR SAMo CNT COM INT NAV ONL PHY PRE PUR UNI"
correlationvector: 8/X76BUyNE+cyd0m.1.0
access-control-allow-headers: Origin, X-Requested-With, Content-Type, Accept
access-control-allow-methods: GET, POST, PUT, DELETE, OPTIONS
access-control-allow-credentials: true
x-frame-options: SAMEORIGIN
x-sitemuse-origin: Azure
x-azure-ref: 0YmNaZAAAAAAienYLqvQ+QLvUSMRBgm5qU0cyRURHRTI4MjIAMWIyYjIyYzEtY2IwYy00OTUwLTg4NzktZDIxMmYyYWM1YTRi
x-edgeconnect-midmile-rtt: 65
x-edgeconnect-origin-mex-latency: 247
date: Tue, 09 May 2023 15:14:42 GMT
set-cookie: MS-CV=8/X76BUyNE+cyd0m.1; domain=.microsoft.com; expires=Wed, 10-May-2023 15:14:42 GMT; path=/;samesite=None; secure
tls_version: tls1.3
strict-transport-security: max-age=31536000
ms-cv: CASMicrosoftCV2e48ec6b.0
ms-cv-esi: CASMicrosoftCV2e48ec6b.0
x-rtag: SMAPROD
```

works for me as well.

**Review by tarunKoyalwar — APPROVED** [review]
@neonbunny  thanks for the PR

```console
$  ./generate-index  
[common/fqdn] Defined 1 items for edgecast
[common/fqdn] Defined 1 items for incapsula
[common/fqdn] Defined 2 items for amazon
[common/fqdn] Defined 4 items for akamai
[common/fqdn] Defined 1 items for cloudflare
[common/fqdn] Defined 1 items for fastly
[cdn] Got 19 items for fastly
[cdn] Got 76 items for google
[cdn] Got 135 items for stackpath
[cdn] Got 142 items for cloudfront
[waf] Got 15 items for cloudflare
[waf] Got 11 items for incapsula
[cloud] Got 600 items for oracle
[cloud] Got 49023 items for azure
[cloud] Got 505 items for zscaler
[cloud] Got 88 items for office365
[cloud] Got 7386 items for aws
[cloud] Got 564 items for google
```

File: cmd/generate-index/provider.yaml (hunk 2 above)
Comment (tarunKoyalwar) [reply cmd/generate-index/provider.yaml:47]: not sure why it was not working earlier . it works now
```console
$ curl -I "https://www.microsoft.com/en-us/download/confirmation.aspx?id=56519" 
HTTP/2 200 
content-type: text/html
cache-control: no-cache, no-store
pragma: no-cache
expires: -1
p3p: CP="ALL IND DSP COR ADM CONo CUR CUSo IVAo IVDo PSA PSD TAI TELo OUR SAMo CNT COM INT NAV ONL PHY PRE PUR UNI"
correlationvector: yZ9QOAtzd0iy98MQ.1.0
access-control-allow-headers: Origin, X-Requested-With, Content-Type, Accept
access-control-allow-methods: GET, POST, PUT, DELETE, OPTIONS
access-control-allow-credentials: true
x-frame-options: SAMEORIGIN
x-sitemuse-origin: Azure
x-azure-ref: 20230526T204913Z-macu5fq3rh3vh9h4quvew7vezc000000009g00000000eeyn
x-edgeconnect-midmile-rtt: 56
x-edgeconnect-origin-mex-latency: 284
date: Fri, 26 May 2023 20:49:13 GMT
set-cookie: MS-CV=yZ9QOAtzd0iy98MQ.1; domain=.microsoft.com; expires=Sat, 27-May-2023 20:49:13 GMT; path=/;samesite=None; secure
tls_version: tls1.3
strict-transport-security: max-age=31536000
ms-cv: CASMicrosoftCV216a2135.0
ms-cv-esi: CASMicrosoftCV216a2135.0
x-rtag: SMAPROD

```

**Review by ehsandeep — APPROVED** [review]

**Pull request merged**